packages/aws-serverless/package.json

```diff
@@ -47,6 +51,9 @@
     "access": "public"
   },
   "dependencies": {
+    "@opentelemetry/instrumentation": "0.48.0",
```

Comment:
m: Instead of doing this, can we export a util/helper from `@sentry/opentelemetry` to register an instrumentation? I thought about this before, and this also makes sense for the docs etc - it means users don't have to install `@opentelemetry/instrumentation` just to add a custom instrumentation, which needs to match the version ideally etc 🤔 

So if we just export an `registerOpenTelemetryInstrumentation(instrumentation)` helper from `@sentry/opentelemetry`, I think that would be a nicer experience?

Reply:
I like it! will make the change

Reply:
Updated!